packages/react-aria-components/test/ComboBox.test.js

```diff
@@ -120,4 +120,32 @@ describe('ComboBox', () => {
     let options = within(groups[0]).getAllByRole('option');
     expect(options).toHaveLength(1);
   });
+
+  // FIXME: not sure why this test hangs
```

Comment:
there were three components that were causing jest to hang. i am not sure why this is happening, but i did create stories for each of those components that demonstrate that render props are indeed working there.

Reply:
Looks like this is a problem. Even in the storybook version, we enter an infinite render loop. You can see this by logging in the Combobox component's main render function.

```
function useTraceUpdate(props) {
  const prev = useRef(props);
  useEffect(() => {
    const changedProps = Object.entries(props).reduce((ps, [k, v]) => {
      if (prev.current[k] !== v) {
        ps[k] = [prev.current[k], v];
      }
      return ps;
    }, {});
    if (Object.keys(changedProps).length > 0) {
      console.log('Changed props:', changedProps);
    }
    prev.current = props;
  });
}
```
and then putting this in Combobox's main render 
```
useTraceUpdate(renderProps);
```
reveals that children are changing each time

I'm not sure of a fix yet for this. Welcome any ideas.

Reply:
Tracking here https://github.com/orgs/adobe/projects/19/views/4?pane=issue&itemId=25889666
for now renderProps on collection components is just unsupported